src/main/java/com/lorepo/icplayer/client/PlayerController.java

```diff
@@ -707,12 +707,6 @@ public void setIframeScroll (int scroll) {
 		}
 	}
 
-	public void handleCurrentPageIdRequest() {
```

Comment:
Domyślam się, że to było zabezpieczenie. Jest kasowane, ale nie rozumiem powodu dlaczego jest kasowane

Reply:
początkowo "owinąłem" metodę gwt metodą javy w celu przygotowania testów, ale finalnie nie było to potrzebne, więc usunąłem